src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,31 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+
+    private static final int MAX_SIZE = 5;
+
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        Set<Character> chars = new TreeSet<>();
+
+        try (FileReader fileReader = new FileReader(fileName);
+                BufferedReader bufferedReader = new BufferedReader(fileReader)) {
```

Comment:
```suggestion
        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
```
FileReader will also be closed